doc/reference/bluetooth/hci.txt

```diff
@@ -800,6 +800,140 @@ When the Set_Scan_Request_Reports command has completed, a Command Complete
+	+--------------------+--------------------------------------+
+
+When the Write_Tx_Power_Level command has completed, a Command Complete
+event shall be generated.
+
+
+Zephyr Read Tx Power Level (per Role/Connection) Command
+========================================================
+
+This command reads the BLE Tx power level. In contrast to the standardized HCI command
+Read_Transmit_Power_Level, which returns the transmitted power level for a specified
+connection handle, this command operates for both connected and unconnected states
+identified by their unique handles.
+
++--------------------------+-------+--------------------+--------------------+
+| Command                  | OCF   | Command            | Return             |
+|                          |       | Parameters         | Parameters         |
++--------------------------+-------+--------------------+--------------------+
+| Read_Tx_Power_Level      | 0x00F | LL_State,          | Status,            |
+|                          |       | Handle             | LL_State,          |
+|                          |       |                    | Handle,            |
+|                          |       |                    | Tx Power Level     |
++--------------------------+-------+--------------------+--------------------+
+
+The Tx power of the BLE radio interface to be read is identified by means of two
+input paraments: the LL State of the desired link and its Handle.
+
+The command returns with an operational Status, the replicated LL State and
+Handle as per the inputs and the current Tx Power Level of the required link.
+
+	Status:                                         Size: 1 Octet
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| 0x00               | Command succeeded                    |
+	| 0x01-0xFF          | Command failed                       |
+	+--------------------+--------------------------------------+
+
+	LL_State:                                       Size: 1 Octet
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| 0x00               | Advertising LL state                 |
+	| 0x01               | Scanning LL state                    |
+	| 0x02               | Connected LL state                   |
+	| 0x03-0xFF          | Reserved (leads to HCI command error)|
+	+--------------------+--------------------------------------+
+
+	Handle:                                        Size: 2 Octets
+	+--------------------+--------------------------------------+
+	| Value              | Parameter Description                |
+	+--------------------+--------------------------------------+
+	| 0x0000 - 0xFFFF    | Handle ID of the state instance to   |
```

Comment:
Please be specific here about what this parameter means for each `TX_Type`. For scanning, it is ignored. For advertising, it's either ignored (no Advertising Extensions) or maps to Advertising Handle (0x00 to 0xEF). For connections, it's Connection Handle (0x0 to 0xEFF).